packages/block-editor/src/components/list-view/leaf.js

```diff
@@ -51,6 +51,7 @@ const ListViewLeaf = forwardRef(
 				level={ level }
 				positionInSet={ position }
 				setSize={ rowCount }
+				disableAriaExpanded={ true }
```

Comment:
This causes the clickable expander to no longer render, which removes the ability to expand / collapse container blocks via the mouse.